### PR TITLE
Rel Notes doc text for 4.6.16 release

### PR DESCRIPTION
**Pull request by jeana-redhat**
For release 2021.02.08
Preview: http://file.rdu.redhat.com/~jrouth/20210205/RNs-4.6.16/release_notes/ocp-4-6-release-notes.html#ocp-4-6-16

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2511,3 +2511,67 @@ link:https://access.redhat.com/solutions/5756401[{product-title} 4.6.15 containe
...
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5780541[{product-title} 4.6.16 container image list]
+
+[id="ocp-4-6-16-features"]
+==== Features
+
+[id="ocp-4-6-16-insights-operator-enhancements"]
+===== Enhancements to Insight Operator
```
Comment (bobfuru) [inline release_notes/ocp-4-6-release-notes.adoc:2530]: Should it be "Insights" like on the next line?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -2511,3 +2511,67 @@ link:https://access.redhat.com/solutions/5756401[{product-title} 4.6.15 containe
...
+
+* The Ironic API service that is embedded during installer-provisioned installation on bare metal now uses four worker nodes instead of eight. This change reduces the RAM usage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1899107[*BZ#1899107*])
+
+* Previously, changing the `serviceAccountIssuer` field of the authentication resource updated the `kubi-apiserver` to validate tokens with the new issuer and reject tokens with the previous issuer. Because `kubi-apiserver` does not support multiple issuers, changing the `serviceAccountIssuer` has the potential to disrupt applications relying on bound tokens. Unless an application is coded to request a new token when the existing token receives `401` responses from the `kubi-apiserver`, the use of an invalid token will continue until the hardware is restarted or the invalid token exceeds 80% of its duration (at which point the kubelet will request a new token).
```
Comment (bobfuru) [inline release_notes/ocp-4-6-release-notes.adoc:2554]: s/duration (at which point the kubelet will request a new token)./duration, at which point the kubelet will request a new token.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -2511,3 +2511,67 @@ link:https://access.redhat.com/solutions/5756401[{product-title} 4.6.15 containe
...
++
+As a workaround, only change the `serviceAccountIssuer` field if disruption is acceptable and restarting all pods is an option.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905573[*BZ#1905573*])
+
+* Previously, the role name was missing on the *create role binding* form. This update presents the role name on the *create role binding* form.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905788[*BZ#1905788*])
+
+* Previously, a low limit for client throttling led to an increasing number of CRDs installed in the cluster. As a result, the requests reaching for API discovery were limited by the client code. This fix increases the limit number by twice of the current amount, thus reducing the frequency of client-side throttling. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906332[*BZ#1906332*])
```
Comment (bobfuru) [inline release_notes/ocp-4-6-release-notes.adoc:2560]: s/by twice of the/to twice the

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -2511,3 +2511,67 @@ link:https://access.redhat.com/solutions/5756401[{product-title} 4.6.15 containe
...
+
+* When using an incorrect user name to verify image signatures, image signature verification is not possible. Using a proper user name results in image signature verification to work as intended. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906796[*BZ#1906796*])
+
+* Previously, if triggers went to Knative Service (KSVC) and In Memory Channel (IMC) from the same broker, Knative resources would not show up in the *Topology* view. This update fixes the NPE so that Knative data is properly returned, which results in the *Topology* view properly showing Knative resources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1907827[*BZ#1907827*])
```
Comment (bobfuru) [inline release_notes/ocp-4-6-release-notes.adoc:2564]: Should you define "NPE" here?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 5]
```diff
@@ -2511,3 +2511,67 @@ link:https://access.redhat.com/solutions/5756401[{product-title} 4.6.15 containe
...
+
+* Previously, init container support was lost during changes to the `oc debug command`. As a result, init containers could not be debugged. This update adds support for init containers in the `oc debug command`, making it possible to debug init containers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913109[*BZ#1913109*])
+
+* Previously, the lack of update in config status during an `Operator sync` execution would not present the most up-to-date (applied) swift configuration. This update fixes the sync process to match config's status with config's spec values. As a result, config status and config spec are now in-sync with the current applied config. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916857[*BZ#1916857*])
```
Comment (bobfuru) [inline release_notes/ocp-4-6-release-notes.adoc:2568]: s/in-sync/in sync

**Review by bobfuru — COMMENTED** [review]
A few thoughts to ponder but otherwise LGTM!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-6-release-notes.adoc:2530]: lol yes thank you

**Pull request merged**